build_library/generate_au_zip.py

```diff
@@ -93,11 +101,11 @@ def _SplitAndStrip(data):
   return return_list
 
 
-def DepsToCopy(ldd_files):
+def DepsToCopy(ldd_files, allow_list):
   """Returns a list of deps for a given dynamic executables list.
     Args:
       ldd_files: List of dynamic files that needs to have the deps evaluated
-      black_list: List of files that we should ignore
+      deny_list: List of files that we should ignore
```

Comment:
This looks wrong - it should be talking about `allow_list`. I suppose that in the past the function has a `black_list` parameter that was later dropped in favor of `BLACK_LIST` global variable, but the docstring wasn't updated.

Reply:
Done